amgut/handlers/open_humans.py

```diff
@@ -75,7 +75,7 @@ class OpenHumansLoginHandler(BaseHandler, OpenHumansMixin):
     _API_URL = urljoin(AMGUT_CONFIG.open_humans_base_url, '/api')
 
     _OAUTH_REDIRECT_URL = urljoin(AMGUT_CONFIG.base_url,
-                                  '/authed/connect/open-humans/')
+                                  'authed/connect/open-humans/')
```

Comment:
ah! nice, thanks

On Mon, Feb 23, 2015 at 8:51 PM, Beau Gunderson <notifications@github.com>
wrote:

> In amgut/handlers/open_humans.py
> <https://github.com/biocore/american-gut-web/pull/375#discussion_r25227067>
> :
>
> > @@ -75,7 +75,7 @@ class OpenHumansLoginHandler(BaseHandler, OpenHumansMixin):
> >      _API_URL = urljoin(AMGUT_CONFIG.open_humans_base_url, '/api')
> >
> >      _OAUTH_REDIRECT_URL = urljoin(AMGUT_CONFIG.base_url,
> > -                                  '/authed/connect/open-humans/')
> > +                                  'authed/connect/open-humans/')
>
> (The reason for that one character change is that later arguments urljoin
> will become absolute to a given domain if they contain a forward slash...
> meaning that:
>
> urljoin('http://foo.com/abc/', '/def/') yields http://foo.com/def/. This
> worked in my testing environment because I had no SITEBASE set.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/american-gut-web/pull/375/files#r25227067>.
>